driver/driver.go

```diff
@@ -39,8 +39,8 @@ type config struct {
 	// One of "run" or "skip".
 	ManualTestMode    string
 	TargetPatternFile string
-	// Expected option: "build"
-	bazelCommandOverride string
+	// Expected value: "build"
```

Comment:
Should this be the following?
```suggestion
	// One of "auto" or "build".
```
The proposed implementation says that if the value is _not_ "build", we treat it as if it were "auto".